server.php

```diff
@@ -125,33 +125,32 @@ function valet_default_site_path($config)
  * Determine the fully qualified path to the site.
  * Inspects registered path directories, case-sensitive.
  */
-$valetSitePath = null;
-$domain = array_slice(explode('.', $siteName), -1)[0];
-
-foreach ($valetConfig['paths'] as $path) {
-    if ($handle = opendir($path)) {
-        while (false !== ($file = readdir($handle))) {
-            if (! is_dir($path.'/'.$file)) continue;
-            if (in_array($file, ['.', '..', '.DS_Store'])) continue;
-
-            // match dir for lowercase, because Nginx only tells us lowercase names
-            if (strtolower($file) === $siteName) {
-                $valetSitePath = $path.'/'.$file;
-                break;
+function get_valet_site_path($valtetConfig, $domain, $siteName)
+{
+    $valetSitePath = null;
+    foreach ($valtetConfig['paths'] as $path) {
+        if ($handle = opendir($path)) {
+            while (false !== ($file = readdir($handle))) {
+                if (! is_dir($path.'/'.$file)) continue;
+                if (in_array($file, ['.', '..', '.DS_Store'])) continue;
+
+                // match dir for lowercase, because Nginx only tells us lowercase names
+                if (strtolower($file) === $siteName || strtolower($file) === $domain) {
+                    $valetSitePath = $path.'/'.$file;
+                }
             }
-            if (strtolower($file) === $domain) {
-                $valetSitePath = $path.'/'.$file;
-                break;
+            closedir($handle);
+            
+            if ($valetSitePath) {
+                return $valetSitePath;
             }
         }
-        closedir($handle);
-        
-        if ($valetSitePath) {
-            break;
-        }
     }
 }
 
+$domain = array_slice(explode('.', $siteName), -1)[0];
+$valetSitePath = get_valet_site_path($valetConfig, $siteName, $domain);
```

Comment:
This doesn't match the function signature: `
function get_valet_site_path($valtetConfig, $domain, $siteName)`

Reply:
Fixed. I wonder why it still worked for me though.